llvm/lib/CodeGen/AsmPrinter/CodeViewDebug.cpp

```diff
@@ -3588,3 +3592,30 @@ void CodeViewDebug::emitDebugInfoForJumpTables(const FunctionInfo &FI) {
     endSymbolRecord(JumpTableEnd);
   }
 }
+
+void CodeViewDebug::emitInlinees(
+    const SmallSet<codeview::TypeIndex, 1> &Inlinees) {
+  // Divide the list of inlinees into chunks such that each chunk fits within
+  // one record.
+  constexpr auto ChunkSize =
+      (MaxRecordLength - sizeof(SymbolKind) - sizeof(int)) / sizeof(int);
```

Comment:
I think `sizeof(uint32_t)` more closely expresses your intent. It's true that int is 32-bits everywhere you can run LLVM, but the reader shouldn't have to think about that.